cleanlab/classification.py

```diff
@@ -602,8 +609,12 @@ def predict(self, *args, **kwargs) -> np.ndarray:
         class_predictions : np.ndarray
           Vector of class predictions for the test examples.
         """
-
-        return self.clf.predict(*args, **kwargs)
+        if self.default_clf:
+            X = force_two_dimensions(args[0])
+            new_args = (X,) + args[1:]
```

Comment:
@jwmueller What was the reasoning behind not specifying `X` as an argument explicitly? (given that that is what sklearn expects). Assuming the features are usually denoted by `X` or `x` we can check for that, but this will become non-trivial to support if users can pass in any arg name.

Reply:
Thanks for catching the error! I think I had the same question while looking at the `predict()` and `predict_proba()` functions. Maybe the reason is if the classifier is defined by the user then it could expect an input dataset name different from `X`